capdl-loader-app/src/main.c

```diff
@@ -1097,7 +1097,7 @@ static void init_tcb(CDL_Model *spec, CDL_ObjID tcb)
     } else {
         ZF_LOGE_IFERR(cdl_cspace_root || cdl_vspace_root || sel4_ipcbuffer,
                       "Could not call seL4_TCB_Configure as not all required objects provided: "
-                      "VSpace: %"PRIxPTR", CSpace: %"PRIxPTR", IPC Buffer: %"PRIxPTR, cdl_vspace_root, cdl_cspace_root, sel4_ipcbuffer);
+                      "VSpace: %p, CSpace: %p, IPC Buffer: %"PRIxPTR, cdl_vspace_root, cdl_cspace_root, sel4_ipcbuffer);
```

Comment:
`sel4_ipcbuffer` is a `seL4_CPtr`, which is a `seL4_Word`, thus `PRIxPTR` is semantically incorrect also. Should be a `SEL4_PRIx_word` then.

Reply:
Thanks, that eluded me.